File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int times = sc.nextInt();

        int quarter;
        int dime;
        int nickel;
        int penny;

        for (int i = 0; i < times; i++) {
            int changes = sc.nextInt();

            quarter = changes / 25;
            changes %= 25;

            dime = changes / 10;
            changes %= 10;

            nickel = changes / 5;

            penny = changes % 5;

            System.out.println(quarter + " " + dime + " " + nickel + " " + penny);
        }
    }
}

File: 백준/Bronze/2720. 세탁소 사장 동혁/세탁소 사장 동혁.cc
#include <iostream>

using namespace std;

int main() {
	
	int n;
	int quarter;
	int dime;
	int nickel;
	int penny;

	cin >> n;

	for (int i = 0; i < n; i++)
	{
		int change;
		cin >> change;

		quarter = change / 25;
		change %= 25;

		dime = change / 10;
		change %= 10;

		nickel = change / 5;
		change %= 5;

		penny = change;

		cout << quarter << " " << dime << " " << nickel << " " << penny << endl;
	}
}